blocks/hero/product-hero.css

```diff
@@ -0,0 +1,137 @@
+
+.hero.block.product .image picture {
+  display: block;
+  position: relative;
+  height: 100%;
+  width: 100%;
+  padding-bottom: 70%;
+}
+
+.hero.block.product .image picture img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  object-position: center bottom;
+  object-fit: cover;
+}
+
+.hero.block.product .content-wrapper {
+  padding: 30px 0;
+  margin: 0 20px;
+}
+
+.hero.block.product .content-wrapper .logo picture {
+  display: block;
+  position: relative;
+  margin-bottom: 20px;
+  height: 100%;
+  width: 100%;
+  padding-bottom: 22%;
+}
+
+.hero.block.product .content-wrapper .logo picture img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  object-position: center center;
+  object-fit: contain;
+}
+
+
+.hero.block.product .title {
+  font-size: var(--body-font-size-xl);
+  line-height: var(--line-height-m);
+  color: var(--black);
+}
+
+.block.hero.gamma-guard-liquid .title {
+  color: var(--gammagard-liq);
+}
+
+.hero.block.product .title p em {
+  font-size: var(--body-font-size-m);
+}
+
+.hero.block.product .references p {
+  margin-bottom: 30px;
+  font-size: var(--body-font-size-xs);
+  line-height: var(--line-height-m);
+}
+
+.hero.block.product .link p.button-container {
+  font-size: var(--body-font-size-m);
+  font-weight: var(--font-weight-semibold);
+  line-height: var(--line-height-m);
+}
+
+.hero.block.product .link .button-container a {
+  display: inline-block;
+  padding: 12px 16px;
+  text-decoration: none;
+  border-radius: 4px;
+  box-shadow: 3px 3px 7px 1px rgba(0 0 0 / 20%);;
+}
+
+.block.hero.gamma-guard-liquid .link .button-container a {
+  color: var(--gammagard-liq);
+}
+
+@media screen and (min-width: 600px) {
+  .hero.block.product .title p {
+    font-size: var(--body-font-size-xxl);
+  }
+
+  .hero.block.product .title p em {
+    font-size: var(--body-font-size-xl);
+  }
+
+  .hero.block.product .link p.button-container {
+    font-size: var(--body-font-size-l);
+  }
+}
+
+@media screen and (min-width: 900px) {
+  .hero.block.product .image {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+  .hero.block.product .image picture {
+    padding-bottom: 0;
+  }
+
+  .hero.block.product .image picture img {
+    object-position: center center;
+  }
+
+  .hero.block.product .content-wrapper {
+    position: relative;
+    max-width: 740px;
+    margin: 0 auto;
+    padding: 0;
+  }
+
+  .hero.block.product .content-wrapper .content {
+    padding: 175px 0;
+    max-width: 500px;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .hero.block.product .content-wrapper {
+    max-width: 980px;
+  }
+}
+
+@media screen and (min-width: 1280px) {
```

Comment:
I noticed a `1280px` breakpoint on the live site when I was working on columns.  Should I go back and refactor to include this new breakpoint?

Reply:
I am not sure i like this tbh. I don't know about keeping it.

Reply:
I think I need to re-eval the max-widths on these images. The reason there's a 1280 breakpoint has to do with the limited sizes on images at different ports. I'm going to re-assess.

Reply:
Awesome.  I'm good with whatever you want to do.  Just let us know when you come to a pattern we should follow.